README.md

```diff
@@ -143,6 +146,10 @@ with the KVM hypervisor driver.
   to use a different command to add `oc` to the _PATH_ environment variable.
   To verify the `oc` version, check the contents of the `~/.minishift/cache/oc` directory.
 
+For general information about using the `oc` command and the command-line console,
+see [Get Started with CLI](https://docs.openshift.org/latest/cli_reference/get_started_cli.html) in
```

Comment:
Actually we have the link already now - https://github.com/minishift/minishift/blob/master/docs/using.md#openshift-client-binary-oc (to the external oc doc). So I think here we should link to our docs in using.md